src/packages/actionsheet/actionsheet.harmony.css

```
@@ -0,0 +1,90 @@
+  color: #c2c4cc !important;
+  cursor: not-allowed;
+}
+.nut-actionsheet-cancel {
+  margin-top: 5px;
+  border-top: 1px solid rgba(0, 0, 0, 0.06);
+  border-radius: 0;
+}
+.nut-actionsheet-safe-area {
+  display: block;
+  width: 100%;
+  padding-bottom: constant(safe-area-inset-bottom);
```

Biome (1.9.4)

[error] Unexpected unknown function: constant

Use a known function instead.
See MDN web docs for more details.

(lint/correctness/noUnknownFunction)